portfolio/src/main/webapp/script.js

```diff
@@ -35,6 +35,30 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+//*********************** PAGE HANDLING ***********************
+/**
+ * Displays the right sidebar
+ */
+let isSidebarOpen = false;
+function toggleSidebar(){
+    let button = document.getElementById("commentbutton");
+    let el = document.getElementById("sidebar");
+    (isSidebarOpen) ? closeSidebar(el, button) : openSidebar(el, button);
```

Comment:
use an if statement instead of a ternary operator if you're not returning anything.